string object_name
---
string success